pyecharts/base.py

```diff
@@ -126,7 +126,9 @@ def _repr_html_(self):
         dom = self._render_notebook_dom_()
         component = self._render_notebook_component_()
         require_config = CURRENT_CONFIG.produce_require_configuration(
-            self._js_dependencies, self._jshost)
+            self._js_dependencies,
+            CURRENT_CONFIG.get_current_jshost_for_jupyter(jshost=self._jshost)
```

Comment:
why not to replace 'CURRENT_CONFIG.get_current_jshost_for_jupyter(jshost=self._jshost)' with self._jshost?